### ISSUE TITLE
A11y_AzureDataStudio_Manage_Name role value :Learn more link in manage dashboard page is not descriptive.

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/azuredatastudio#22656]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp;  #Win32; #WCAG4.1.2; #Name Role Value; #Screen Reader;  #Benchmark; #AILimited;  #NVDA; #JAWS; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; 

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**
### Environment and OS details:
Azure Data Studio Version: 1.42.0 (user setup)
Windows 11 Enterprise Version	22H2
Installed on	‎28-‎12-‎2022
OS build	22621.1413
Experience	Windows Feature Experience Pack 1000.22639.1000.0
### Pre-Requisite:

1. Install Azure data studio From here https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16&tabs=redhat-install%2Credhat-uninstall
2. Once installation done , Install MySQL Extension in Azure data studio
3. Tab Till  new connection and press enter 
4. Enter a valid server name username and password and press connect
### Repro Steps:

1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment"
4. Tab till server 
5. Tab till manage and press enter
6. Tab till Learn more is descriptive or not.
### Actual Result:
Learn more link in manage dashboard page is not descriptive.

### Expected Result:
Learn more link in manage dashboard page should be descriptive.
### Attachment:

![Screenshot (79)](https://user-images.githubusercontent.com/100189407/229701638-056b8d4a-b8ab-4852-9257-b5e9114a6542.png)
![Screenshot (87)](https://user-images.githubusercontent.com/100189407/229814569-5c214840-d103-4006-97c9-060ed74a2439.png)

**Comment by VijayaBharathyA** [comment]
Primary bug : https://github.com/microsoft/azuredatastudio-mysql/issues/188

**Comment by alanrenmsft** [comment]
@VijayaBharathyA what is the expected behavior? what exactly need to be added.

**Comment by VijayaBharathyA** [comment]
#Regressed:05-15-23; #Closed
### Verified issue in below environment :
Issue is fixed
Version: 1.45.0-insider (user setup)
Commit: 4329d04da3b86689556d4dceab7d9b3d5cde3f58
Date: 2023-05-15T05:30:26.887Z
VS Code: 1.70.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
![Screenshot (153)](https://github.com/microsoft/azuredatastudio/assets/100189407/5302f4fb-d042-46fa-9beb-cf94d359569d)
![Screenshot (152)](https://github.com/microsoft/azuredatastudio/assets/100189407/c717d227-400a-41cc-8c64-52a049517474)
